src/main/telemetry/smartport.c

```diff
@@ -710,6 +712,8 @@ void handleSmartPortTelemetry(void)
                         tmpi += 1000;
                     if (STATE(GPS_FIX_HOME))
                         tmpi += 2000;
+                    if (ARMING_FLAG(ARMED) && IS_RC_MODE_ACTIVE(BOXHOMERESET) && !FLIGHT_MODE(NAV_RTH_MODE) && !FLIGHT_MODE(NAV_WP_MODE))
+                        tmpi += 4000;
```

Comment:
This is not entirely correct. Home reset does it's job only **once** when mode is activated.

Reply:
When home reset is triggered, this value is set and sent to my transmitter via telemetry. I tested this branch tonight and it worked flawlessly (it was the most boring flight ever BTW as it as 10pm and I only flew about 10 feet).

Also, the same logic existed in a previous RC build which also worked. I simply moved it to a different (and more appropriate) sensor.